app/components/candidate_interface/referees_review_component.rb

```diff
@@ -80,6 +80,36 @@ def feedback_status_row(referee)
           value: feedback_status_label(referee) +
             content_tag(:p, t('application_form.referees.info.not_requested_yet'), class: 'govuk-body govuk-!-margin-top-2'),
         }
+      elsif referee.feedback_refused?
+        {
+          key: 'Status',
+          value: feedback_status_label(referee) +
+            content_tag(:p, t('application_form.referees.info.declined'), class: 'govuk-body govuk-!-margin-top-2'),
+        }
+      elsif referee.feedback_overdue?
+        {
+          key: 'Status',
+          value: feedback_status_label(referee) +
+            content_tag(:p, t('application_form.referees.info.feedback_overdue'), class: 'govuk-body govuk-!-margin-top-2'),
+        }
+      elsif referee.feedback_requested? && referee.requested_at > Time.zone.now - 5.days
+        {
+          key: 'Status',
+          value: feedback_status_label(referee) +
+            content_tag(:p, t('application_form.referees.info.awaiting_reference_sent_less_than_5_days_ago'), class: 'govuk-body govuk-!-margin-top-2'),
+        }
+      elsif referee.feedback_requested?
+        {
+          key: 'Status',
+          value: feedback_status_label(referee) +
+            content_tag(:p, t('application_form.referees.info.awaiting_reference_sent_more_than_5_days_ago'), class: 'govuk-body govuk-!-margin-top-2'),
+        }
+      elsif referee.cancelled?
+        {
+          key: 'Status',
+          value: feedback_status_label(referee) +
+            content_tag(:p, t('application_form.referees.info.awaiting_reference_sent_more_than_5_days_ago'), class: 'govuk-body govuk-!-margin-top-2'),
+        }
```

Comment:
Minor nit: would it be better to if/else on the value of this to DRY up a bit?

```ruby
def feedback_status_row(referee)
  value = feedback_status_label(referee) + if else if else if else end

  {
    key: 'Status',
    value: value,
  }
end
```

Reply:
👍 have done